redbot/cogs/audio/core/commands/controller.py

```diff
@@ -733,25 +733,15 @@ async def command_volume(self, ctx: commands.Context, vol: int = None):
                 description=_("You need the DJ role to change the volume."),
             )
 
-        if vol < 0:
-            vol = 0
-        if vol > 150:
-            vol = 150
-            await self.config.guild(ctx.guild).volume.set(vol)
-            if self._player_check(ctx):
-                player = lavalink.get_player(ctx.guild.id)
-                await player.set_volume(vol)
-                player.store("channel", ctx.channel.id)
-                player.store("guild", ctx.guild.id)
-        else:
-            await self.config.guild(ctx.guild).volume.set(vol)
-            if self._player_check(ctx):
-                player = lavalink.get_player(ctx.guild.id)
-                await player.set_volume(vol)
-                player.store("channel", ctx.channel.id)
-                player.store("guild", ctx.guild.id)
+        vol = max(0, vol)
```

Comment:
I suppose that volume over 150 should still be disallowed here?
```suggestion
        vol = max(0, min(vol, 150))
```

Reply:
This is an old restriction from when this was the cap. 

The volume can go above 150% and has been able to for a very long time, this just removed the artificial restriction.